pkg/apis/servicecatalog/testing/fuzzer.go

```diff
@@ -90,97 +89,9 @@ func createPlanMetadata(c fuzz.Continue) (*runtime.RawExtension, error) {
 	return &runtime.RawExtension{Raw: b}, nil
 }
 
-// FuzzerFor can randomly populate api objects that are destined for version.
```

Comment:
Did some of these (ie, TypeMeta) move into the generic fuzzer?

Reply:
@pmorie yes. the `FuzzerFor`method is replaced with `func servicecatalogFuncs(codecs runtimeserializer.CodecFactory) []interface{}` (see below), and now we can combine fuzzer functions from different packages (see at the end of this file):
```go
 // FuzzerFuncs provides merged set of fuzzers from different sources
 var FuzzerFuncs = fuzzer.MergeFuzzerFuncs(
 	genericfuzzer.Funcs,
 	servicecatalogFuncs,
 )
```
so I left only Service Catalog specific types there.